src/set_theory/surreal/basic.lean

```diff
@@ -219,16 +219,10 @@ end
 
 end pgame
 
-/-- The equivalence on numeric pre-games. -/
-def surreal.equiv (x y : {x // pgame.numeric x}) : Prop := x.1.equiv y.1
-
 open pgame
 
-instance surreal.setoid : setoid {x // pgame.numeric x} :=
-⟨λ x y, x.1 ≈ y.1,
- λ x, equiv_rfl,
- λ x y, pgame.equiv.symm,
- λ x y z, pgame.equiv.trans⟩
+/-- The equivalence on numeric pre-games. -/
+instance surreal.setoid : setoid numeric := by apply_instance
```

Comment:
Do we still need this instance now that it can be automatically inferred?

Reply:
I could write `def surreal := quotient (by apply_instance : setoid numeric)` if preferable.

Reply:
The setoid instance allows us to use `⟦⟧` which may be useful.

Reply:
Nevermind, the instance could already be inferred.